### PR TITLE
⬆️ Update submodule

### PR DESCRIPTION
**Pull request by kirkkwang**
This commit will update the submodule to mainly bring in the ruby-saml patch.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1242,7 +1242,7 @@ GEM
       multipart-post
       oauth2
     ruby-progressbar (1.13.0)
-    ruby-saml (1.16.0)
+    ruby-saml (1.17.0)
```
Comment (ShanaLMoore) [inline Gemfile.lock:1245]: Gemfile.lock shouldn't exist in the knapsack, according to Rob

Comment (kirkkwang) [reply Gemfile.lock:1245]: alright sounds good i can delete the whole thing then

**Comment by aprilrieger** [comment]
@kirkkwang can this be closed?

**Comment by kirkkwang** [comment]
> @kirkkwang can this be closed?

Not yet, I need an approval and it has to get merged.

**Review by aprilrieger — APPROVED** [review]

**Pull request merged**